transactions/stx.ts

```diff
@@ -454,4 +457,21 @@ export async function generateContractDeployTransaction(options: {
   }
 }
 
+export async function getLatestNonce(stxAddress: string, network: SettingsNetwork): Promise<LatestNonceResponse> {
+  const baseUrl = network?.address;
+  const apiUrl = `${baseUrl}/extended/v1/address/${stxAddress}/nonces`;
```

Comment:
it will probably throw an axios error even with the optional chaining operator `?.` since the apiUrl will be malformed

maybe we just rely on the type checking, since network is a required param
```suggestion
  const baseUrl = network.address;
  const apiUrl = `${baseUrl}/extended/v1/address/${stxAddress}/nonces`;
```